Assets/MixedRealityToolkit.SDK/Experimental/Features/Dwell/DwellHandler.cs

```diff
@@ -0,0 +1,210 @@
+﻿// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License. See LICENSE in the project root for license information.
+
+using Microsoft.MixedReality.Toolkit.Input;
+using System;
+using UnityEngine;
+using UnityEngine.Events;
+
+namespace Microsoft.MixedReality.Toolkit.Experimental.Dwell
+{
+    [Serializable]
+    public class PointerUnityEvent : UnityEvent<IMixedRealityPointer>
```

Comment:
Maybe we can do that when this feature is upgraded to be part of regular namespace.

Reply:
Note that there already is a PointerUnityEvent, so when that day happens we'd still have to reconcile that.

If you want to avoid that conversation you can also go DwellUnityEvent (which names it after the situations in which it's used, rather than what it represents). It's still not GREAT exactly but could be a reasonable tradeoff

Reply:
If there is already an existing PointerUnityEvent, can we just use that? Or is it not a UnityEvent<IMixedRealityPointer>?

Reply:
The existing UnityPointerEvent is public class PointerUnityEvent : UnityEvent<**MixedRealityPointerEventData**> { } 
so it is a different type. i will rename this one, and put it in a separate to follow MRTK coding guidelines.